README.md

```diff
@@ -1,36 +1,89 @@
 # Linea
 
+### Linea is a developer-ready Layer 2 network, scaling Ethereum by providing an Ethereum-equivalent environment in which to execute transactions, which are then submitted to Ethereum Mainnet through a **zero-knowledge rollup**. 
+
+# These docs
+
 This documentation repo is built using [Docusaurus 2](https://docusaurus.io/).
 
 See [more](https://docs-template.consensys.net/) information about using Docusaurus quickly.
 
-### Local development
+## Contributing to our docs
+See something missing? Error in our documentation? Create an issue [here](https://github.com/ConsenSys/doc.zk-evm/issues).
+
+Alternatively, help us improve our documentation! [Fork our repo](https://github.com/ConsenSys/doc.zk-evm/fork), create a pull request, and tag us for review! (for help on this, see below)
+
+You can find our contribution guidelines [here](https://docs-template.consensys.net/category/getting-started) and our style guide [here](https://docs-template.consensys.net/getting-started/style-guide).
+
+Take a look at some [good first issues](https://github.com/ConsenSys/doc.zk-evm/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) to get started.
+
+### How to submit a suggestion or change
+The best way to suggest a change to these docs is through a process known as a **pull request**. If you're not familiar with how that works, check out [GitHub's guide here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request).
+
+If that process is too involved for you, you can always open a thread on the [Community forum](https://community.linea.build/), or a ticket on the [Support page](https://support.linea.build/hc/en-us). 
+
+If you **are** familiar with making a Pull Request, we **highly recommend that you run a version of these docs locally, and preview your changes locally, before submitting them**. In fact, it's part of the PR process.
```

Comment:
I think we should make pull request lowercase for consistency with the rest of the page

Reply:
big brain